test/minters/fixed-price/ZoraCreatorFixedPriceSaleStrategy.t.sol

```diff
@@ -323,12 +338,16 @@ contract ZoraCreatorFixedPriceSaleStrategyTest is Test {
         address tokenRecipient = address(322);
         vm.deal(tokenRecipient, 20 ether);
 
+        uint256 totalReward = computeTotalReward(1);
+
         vm.startPrank(tokenRecipient);
         vm.expectRevert(abi.encodeWithSignature("WrongValueSent()"));
-        target.mint{value: 0.9 ether}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""));
+        target.mint{value: 0.9 ether + totalReward}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""), address(0), address(0));
+
         vm.expectRevert(abi.encodeWithSignature("WrongValueSent()"));
-        target.mint{value: 1.1 ether}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""));
-        target.mint{value: 1 ether}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""));
+        target.mint{value: 1.1 ether + totalReward}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""), address(0), address(0));
+
+        target.mint{value: 1 ether + totalReward}(fixedPrice, newTokenId, 1, abi.encode(tokenRecipient, ""), address(0), address(0));
```

Comment:
all of the changing of the tests here to add empty address fields seems like a bit of an antipattern, like these arguments shouldn't exist on the main `mint` function if they're not always used.